caliper_sender.py

```diff
@@ -104,7 +104,7 @@ def send_caliper_event():
     if len(batch) != 0:
         send_event_batch(batch)
 
-def caliper_page_event(event):
+def get_caliper_event(event, event_type):
```

Comment:
This is an interesting idea of passing the event as a parameter. I think having an optional "Action" might be useful, even though it isn't for either of these events. 

ViewEvent only has one action (Viewed) and NavigationEvent (NavigatedTo) 

However some of these may have more actions. Maybe if this was more forward thinking it would be this? Then in each type check
 
if not event_action:
   event_action = (Default Action, like "NavigatedTo")

```suggestion
def get_caliper_event(event, event_type, event_action=None):
```

Reply:
Or maybe don't make it optional and just pass the action in, that would work too. ;)

Reply:
Should this event_action also be related to the `act` field in the `useinfo` table? Currently for the `page` event there's only `view` act, but I'm not sure if there'll be more act types.

Reply:
Something is going to have to determine the caliper event mapping. Whether it's the send_caliper_event or get_caliper_event I don't think it makes a difference. 

I think the current way you have it the easiest would just be

```
            if event.get('event') == 'page' and event.get('act') == 'view':
                caliper_event = get_caliper_event(event, "ViewEvent", action="Viewed")
```

If we had other events and actions they may need to call this method differently.